### PR TITLE
Added device compatibility page

### PR DESCRIPTION
**Pull request by pwojnaro**
This PR, when merged, will add compatible devices reference guide. The page covers iOS and Android compatibility with Routegy QR and NFC touchpoints.

### CODE REVIEW

File: app/docs/reference/index.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ If you're currently asking yourself a "How does this work..." question, you're i
 * [Activity Streams & Feeds](/reference/activity/)
 * [Authentication & Authorization](/reference/auth/)
 * [Codes](/reference/codes/)
+* [Compatible Devices](/reference//compatible-devices/)
```
Comment (ahawker) [inline app/docs/reference/index.md:18]: Remove the extra slash after `reference`; breaks linking

File: app/docs/reference/compatible-devices.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+title: "Compatible devices | Reference"
+description: "Lists devices compatible with Routegy QR codes and NFC tags"
+sidebar: false
+---
+
+# Devices compatible with QR and NFC touchpoints
+
+Interacting with Routegy QR and NFC touchpoints relies on built-in capabilities of customer's device. These capabilities are now a standard feature in most modern devices that run either iOS or Android.
+
+
```
Comment (ahawker) [inline app/docs/reference/compatible-devices.md:11]: Remove extra newline

File: app/docs/reference/compatible-devices.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Compatible devices | Reference"
+description: "Lists devices compatible with Routegy QR codes and NFC tags"
```
Comment (ahawker) [inline app/docs/reference/compatible-devices.md:3]: I've been trying to form these as questions relative to their section. For reference, these are "how does X work".

I think for this, i'd recommend: `How do I know if my device is compatible?`

File: app/docs/reference/compatible-devices.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+sidebar: false
+---
+
+# Devices compatible with QR and NFC touchpoints
+
+Interacting with Routegy QR and NFC touchpoints relies on built-in capabilities of customer's device. These capabilities are now a standard feature in most modern devices that run either iOS or Android.
```
Comment (ahawker) [inline app/docs/reference/compatible-devices.md:9]: Would change to: "...most modern Android and iOS devices."

File: app/docs/reference/compatible-devices.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+
+## iOS device compatibility
+
+QR code reading capability is built into the camera app on any iPhone, iPad and iPod that runs iOS 11 or newer.
```
Comment (ahawker) [inline app/docs/reference/compatible-devices.md:14]: I'd bold the version so it pops a bit more, e.g. **iOS 11**. Same below with **iOS 13** and the other places where you call the Android OS versions as well.

File: app/docs/reference/compatible-devices.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+NFC tag reading capability is supported in iOS 13 (and newer) on the following devices: iPhone XS, XS Max, XR, 11, 11 Pro and 11 Pro Max.
+
+## Android device compatibility
+
+QR code reading capability is built into the camera app on any device that runs Android 9 or newer. In addition to this, devices that run Android 8 can scan QR codes by using Google Lens app.
```
Comment (ahawker) [inline app/docs/reference/compatible-devices.md:20]: Would remove "In addition to this.." just started with "Devices that run Android 8..."

File: app/docs/reference/compatible-devices.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+NFC tag reading is built into most modern Android devices. You can check if your devices supports NFC by going into _Settings->Connected devices_ (or a similar option like _Wireless & networks_ or _Connections_). If _NFC_ is present in these settings,your device supports reading NFC tags (as long as it is turned on).
```
Comment (ahawker) [inline app/docs/reference/compatible-devices.md:22]: 1) "devices supports". Device should be singular.

2) "settings,your" - Add space after comma.

File: app/docs/reference/compatible-devices.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Compatible devices | Reference"
```
Comment (trentpeterson) [inline app/docs/reference/compatible-devices.md:2]: -> Device Compatibility

File: app/docs/reference/compatible-devices.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+description: "How do I know if my device is compatible with Routegy touchpoints?"
+sidebar: false
+---
+
+# Devices compatible with QR and NFC touchpoints
+
+Interacting with Routegy QR and NFC touchpoints relies on built-in capabilities of customer's device. These capabilities are now a standard feature in most modern iOS and Android devices.
```
Comment (trentpeterson) [inline app/docs/reference/compatible-devices.md:9]: relies on -> uses
customer's device -> a person's device
are now a -> are a

File: app/docs/reference/compatible-devices.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+## iOS device compatibility
+
+QR code reading capability is built into the camera app on any iPhone, iPad and iPod that runs **iOS 11** or newer.
```
Comment (trentpeterson) [inline app/docs/reference/compatible-devices.md:13]: The capability to scan QR codes is built into the camera app on iPhones, iPads, and iPods that run **iOS 11** or newer.

File: app/docs/reference/compatible-devices.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+
+NFC tag reading capability is supported in **iOS 13** (and newer) on the following devices: iPhone XS, XS Max, XR, 11, 11 Pro and 11 Pro Max.
```
Comment (trentpeterson) [inline app/docs/reference/compatible-devices.md:15]: The capability to scan NFC tags is supported...

(and newer) -> and up

11 Pro and -> 11 Pro, and

File: app/docs/reference/compatible-devices.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+
+## Android device compatibility
+
+QR code reading capability is built into the camera app on any device that runs **Android 9** or newer. Devices that run **Android 8** can scan QR codes by using Google Lens app.
```
Comment (trentpeterson) [inline app/docs/reference/compatible-devices.md:19]: The capability to scan QR codes is built into the camera app on devices that run...

using Google -> using the Google

File: app/docs/reference/compatible-devices.md [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+
+NFC tag reading is built into most modern Android devices. You can check if your device supports NFC by going into _Settings->Connected devices_ (or a similar option like _Wireless & networks_ or _Connections_). If _NFC_ is present in these settings, your device supports reading NFC tags (as long as it is turned on).
```
Comment (trentpeterson) [inline app/docs/reference/compatible-devices.md:21]: The ability to scan NFC tags is...

(as long as it is turned on) -> as long as it is turned on

File: app/docs/reference/compatible-devices.md [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+
+Here is a list of some of the popular devices that support this capability:
```
Comment (trentpeterson) [inline app/docs/reference/compatible-devices.md:23]: Here is a list of popular devices

File: app/docs/reference/compatible-devices.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+* Google  Pixel, Pixel 2, Pixel 3, Pixel 3a and all of their XL versions
+* Samsung Galaxy S8, Galaxy S8+, Galaxy S9, Galaxy S9+ and Galaxy S10, S10 G5, S10
```
Comment (trentpeterson) [inline app/docs/reference/compatible-devices.md:25]: get rid of and

File: app/docs/reference/compatible-devices.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+NFC tag reading capability is supported in **iOS 13** (and newer) on the following devices: iPhone XS, XS Max, XR, 11, 11 Pro and 11 Pro Max.
+
+## Android device compatibility
+
+QR code reading capability is built into the camera app on any device that runs **Android 9** or newer. Devices that run **Android 8** can scan QR codes by using Google Lens app.
+
+NFC tag reading is built into most modern Android devices. You can check if your device supports NFC by going into _Settings->Connected devices_ (or a similar option like _Wireless & networks_ or _Connections_). If _NFC_ is present in these settings, your device supports reading NFC tags (as long as it is turned on).
+
+Here is a list of some of the popular devices that support this capability:
+* Google  Pixel, Pixel 2, Pixel 3, Pixel 3a and all of their XL versions
```
Comment (trentpeterson) [inline app/docs/reference/compatible-devices.md:24]: 3a and -> 3a, and
and all of their -> and their corresponding

File: app/docs/reference/compatible-devices.md [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+* Samsung Galaxy S8, Galaxy S8+, Galaxy S9, Galaxy S9+ and Galaxy S10, S10 G5, S10
+* LG G8, G8s, Q9, Q8, Q7, G7 ThinQ
+* HTC U12+, U11 Eyes, U11/Life/+
+* Motorola Moto Z3 Play, Moto X4
+* Huawei P10, P10 Plus, P10 Lite, P20, P20 Plus, P20 Lite, P30, P30 Plus and P30 Lite
```
Comment (trentpeterson) [inline app/docs/reference/compatible-devices.md:29]: get rid of and

File: app/docs/reference/compatible-devices.md [hunk 18]
```diff
@@ -0,0 +1,31 @@
...
+* OnePlus 6, T5, 5, 3T, 3, One
+* Xiaomi Mi6, Mi6X, Mi8, Mi8 Pro, Mi9 and Mi9 SE
```
Comment (trentpeterson) [inline app/docs/reference/compatible-devices.md:31]: get rid of and

File: app/docs/reference/index.md [hunk 19]
```diff
@@ -15,6 +15,7 @@ If you're currently asking yourself a "How does this work..." question, you're i
 * [Activity Streams & Feeds](/reference/activity/)
 * [Authentication & Authorization](/reference/auth/)
 * [Codes](/reference/codes/)
+* [Compatible Devices](/reference/compatible-devices/)
```
Comment (trentpeterson) [inline app/docs/reference/index.md:18]: Device Compatibility

**Pull request merged**